webapp/components/NotificationsTable/NotificationsTable.spec.js

```diff
@@ -3,7 +3,7 @@ import { config, mount, RouterLinkStub } from '@vue/test-utils'
 import Vuex from 'vuex'
 import NotificationsTable from './NotificationsTable'
 
-import { notifications } from '~/components/utils/Notifications'
+import { testNotifications } from '~/components/utils/Notifications'
```

Comment:
don't understand this change...

Reply:
Here the change is not necessary, but at the moment in the file `NotificationsTable.story.js` it can not be named `notifications`, because there this name is already defined.